binaries/workerserver/main.go

```diff
@@ -91,6 +92,19 @@ func main() {
 		func() execer.Memory {
 			return execer.Memory(*memCapFlag)
 		},
+		func() server.DriveTypeFunc {
```

Comment:
If this passes for internal use as well, could define once as a default in setup.go (or if not in default but we want to reuse, throw under common somewhere)